.packit.yaml

```diff
@@ -16,8 +16,10 @@ actions:
 jobs:
   - job: copr_build
     trigger: pull_request
+    notifications:
+      failure_comment:
+        message: "Ephemeral COPR build failed. @lsm5 please check."
```

Comment:
Can you create a GitHub Team or Organization and use that instead?  Even if that group only has you in it.

Reply:
hmm, don't know if the packit feature works for teams but we can try. We'll also need something similar for the revdep failure tag @martinpitt

Reply:
It's not a feature of packit. You can specify whichever text you want in the notification. The `@user` is a GitHub feature. But I'm not sure if e.g. tagging the @cockpit-project org actually works (that would be way too big, just for testing -- if it works, sorry everyone for the noise!)

Reply:
done

Reply:
@martinpitt maybe a cockpit team with the three of you would work? I created @containers/packit-build just now.

Reply:
@TomSweeneyRedHat cockpit org/team can be handled in a separate PR.

Reply:
@martinpitt i could even create a `containers/cockpit-tests` team if that works for you.

Reply:
Yup, I'm happy to handle the cockpit notifications separately. But I'm still not convinced that tagging a team actually works -- did anyone from the cockpit-project org actually get this notification? Please react with :+1: here if you did. Thanks

Reply:
@martinpitt I see both team and org mentions in your latest comment. To be clear, I haven't created any cockpit team in the containers org yet.

Reply:
@lsm5 I just asked my colleagues in @cockpit-project, and they did *not* get the notification (that also makes sense -- spam protection). So an org won't work. A team might, let me test that in https://github.com/cockpit-project/bots/pull/5288#issuecomment-1731373705

Reply:
Team tagging indeed seems to work :tada: